modules/rostests/apitests/ntdll/NtApphelpCacheControl.c

```diff
@@ -340,12 +341,21 @@ START_TEST(NtApphelpCacheControl)
     char **argv;
     int argc;
 
+    // Soft get, check and skip, intead of hard load error when executed on NT < 5.2 API (like Windows XP).
+    pNtApphelpCacheControl = (void*)GetProcAddress(GetModuleHandleA("ntdll.dll"), "NtApphelpCacheControl");
+    if (!pNtApphelpCacheControl)
+    {
+        win_skip("NtApphelpCacheControl (NT >= 5.2 API) not available, skipping tests\n");
+        return;
+    }
+
     pRegisterServiceCtrlHandlerExA = (void*)GetProcAddress(GetModuleHandleA("advapi32.dll"), "RegisterServiceCtrlHandlerExA");
     if (!pRegisterServiceCtrlHandlerExA)
     {
         win_skip("RegisterServiceCtrlHandlerExA not available, skipping tests\n");
         return;
     }
+
```

Comment:
Why?

Reply:
A blank line after a `return` is usually more readable. Don't you want it?

Reply:
As a general rule, we don't want unnecessary changes.